modules/flowable-http-common/src/main/java/org/flowable/http/common/impl/apache/client5/ApacheHttpComponents5FlowableHttpClient.java

```diff
@@ -170,6 +170,7 @@ public AsyncExecutableHttpRequest prepareRequest(HttpRequest requestInfo) {
                 }
                 case "DELETE": {
                     request = AsyncRequestBuilder.delete(uri);
+                    setRequestEntity(requestInfo,request);
```

Comment:
Nitpick to match project formatting style; add a space
```suggestion
                    setRequestEntity(requestInfo, request);
```

Reply:
formatted sources properly.

as a side note, the Eclipse formatter settings from https://github.com/flowable/flowable-engine/tree/main/ide-settings/eclipse often change existing source files when applying formatter (hence trying to manually match formatting rules for now) - the formatter rules do not define a maximum line width at all, and some indentations for broken lines differ from what (many?) of the source files actually seem to be using..